airbyte-commons/src/main/java/io/airbyte/commons/resources/MoreResources.java

```diff
@@ -81,12 +82,15 @@ public static String readResource(String name) throws IOException {
   }
 
   @SuppressWarnings("UnstableApiUsage")
-  public static void writeResource(String filename, String contents) {
-    final Path source = Paths.get(Resources.getResource("").getPath());
+  public static String writeTmpFile(String filename, String contents) {
```

Comment:
This was throwing errors for me when I ran it using Intellij. The classloader was attempting to write to the JUnit Platform jar instead of a proper file location.

Ideally, I would split this out to a separate PR. However it would involve cleaning this one up, and would affect my development work on this PR. I don't think this change is complex enough to worth splitting it out. Thus, I'm going to combine these together. I've tried my best to isolate the changes by squashing all the MoreResources related changes into the same commit. Thanks in advance, reviewers!

Reply:
as discussed this change is fine but the new version of this method shouldn't be in this class.

Reply:
Done!